src/chrome-launcher.ts

```diff
@@ -197,6 +202,11 @@ class Launcher {
     this.outFile = this.fs.openSync(`${this.userDataDir}/chrome-out.log`, 'a');
     this.errFile = this.fs.openSync(`${this.userDataDir}/chrome-err.log`, 'a');
 
+    // set preferences
+    if (Object.keys(this.prefs).length) {
+      this.setBrowserPrefs()
```

Comment:
nit: if we validate here, let's pass the validated prefs in, otherwise move validation to setBrowserPrefs :)